sonic-slave-buster/Dockerfile.j2

```diff
@@ -305,7 +305,18 @@ RUN apt-get update && apt-get install -y \
         libxml2-utils \
         xsltproc \
         python-lxml \
-        libexpat1-dev
+        libexpat1-dev \
+# For WPA supplication
+        libssl-dev           \
```

Comment:
We might need to check the dependencies here. In our build, we don't have extra packages needed specifically for wpa-supplicant..And in the list. libdbus-1-dev, libncurses5-dev, libreadline-dev are already in the list from other components.

Reply:
My understanding is that we should list all dependencies for a component so that this block is isolated.
It means if some dependencies of other component were modified, like delete some dependencies, there is no effect to wpa supplicant so that we don't need to add those deleted dependencies to this block of wpa supplicant.

Reply:
Better to remove this block if not really necessary.